src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx

```diff
@@ -1383,8 +1412,8 @@ Here's a sample configuration:
             CrossNewRelic.Current.TrackShellNavigatedEvents();
 
             // Set optional agent configuration
-            // Options are: crashReportingEnabled, loggingEnabled, logLevel, collectorAddress, crashCollectorAddress,analyticsEventEnabled, networkErrorRequestEnabled, networkRequestEnabled, interactionTracingEnabled,webViewInstrumentation, fedRampEnabled
-            AgentStartConfiguration agentConfig = new AgentStartConfiguration(crashReportingEnabled:false);
+            // Options are: crashReportingEnabled, loggingEnabled, logLevel, collectorAddress, crashCollectorAddress,analyticsEventEnabled, networkErrorRequestEnabled, networkRequestEnabled, interactionTracingEnabled,webViewInstrumentation, fedRampEnabled,offlineStorageEnabled
+            AgentStartConfiguration agentConfig = new AgentStartConfiguration(crashReportingEnabled:false,offlineStorageEnabled:false);
```

Comment:
```suggestion
            AgentStartConfiguration agentConfig = new AgentStartConfiguration(crashReportingEnabled: false, offlineStorageEnabled: false);
```
@ndesai-newrelic Should this have spaces?

Reply:
user can put that.